<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATFTuningClass</class>
 <widget class="QWidget" name="ATFTuningClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ATFTuning</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1561</width>
     <height>1031</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="algo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>algo</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_set">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>29</y>
       <width>850</width>
       <height>961</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="effect_label">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>effect</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_enable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>730</y>
       <width>701</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="default_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>default</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_init">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>700</width>
       <height>691</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FS">
    <attribute name="title">
     <string>fs-calc</string>
    </attribute>
    <widget class="QTableWidget" name="flicker_config">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>751</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="flicker_calc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>1051</width>
       <height>581</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="default_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>config</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="default_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>effect</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>1030</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>1030</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <zorder>pushButton_load</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ATFTuning.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked()</signal>
   <receiver>ATFTuningClass</receiver>
   <slot>on_pushButton_loadParam_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>1052</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>ATFTuningClass</receiver>
   <slot>on_pushButton_saveParam_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>1052</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_loadParam_clicked()</slot>
  <slot>on_pushButton_saveParam_clicked()</slot>
 </slots>
</ui>
